src/widgets/forms/form-view.component.tsx

```diff
@@ -53,17 +75,50 @@ const FormView: React.FC<FormViewProps> = ({
         onClick={() => launchFormEntry(form)}
         className={styles.customCheckBoxContainer}
       >
-        <CheckmarkFilled16 />
+        {form.checked ? <CheckmarkFilled16 /> : <RadioButton16 />}
         <div className={styles.label}>{label}</div>
       </div>
     );
   };
 
   return (
     <div className={styles.formContainer}>
-      {forms.map((form, index) => (
-        <CheckedComponent key={index} label={form.name} form={form} />
-      ))}
+      <Search
+        id="searchInput"
+        labelText=""
+        className={styles.formSearchInput}
+        placeholder={t("searchForForm", "Search for a form")}
+        onChange={evnt => handleSearch(evnt.target.value)}
+      />
+      <>
+        {!isEmpty(searchTerm) && !isEmpty(allForms) && (
+          <p className={styles.formResultsLabel}>
+            {allForms.length} {t("matchFound", "match found")}
+          </p>
+        )}
+        {isEmpty(allForms) && !isEmpty(searchTerm) && (
+          <Tile light className={styles.formTile}>
+            <EmptyDataIllustration />
+            <p className={styles.content}>
+              {t(
+                "sorryNoFormsHaveBeenFound",
+                "Sorry, no forms have been found"
+              )}
+            </p>
+            <p className={styles.action}>
+              {t(
+                "trySearchingForTheFormUsingAnAlternativeNameorKeyword",
```

Comment:
```suggestion
                "formSearchHint",
```

Reply:
(your names were good, don't worry about this kind of thing in the future. This is a very minor improvement and coming up with good names for things is often very hard)